src/cadenza.js

```diff
@@ -165,6 +172,8 @@ export class CadenzaClient {
    * @param {GlobalId} [options.highlightGlobalId] - The ID of an item to highlight / expand in the navigator
    * @param {string} [options.mediaType] - Set to 'application/pdf' for views of type "JasperReports report"
    *     to show the report PDF directly, without any Cadenza headers or footers.
+   * @param {string} [options.operationMode] - Whether to enable simplified operation mode
```

Comment:
Proposal: Use a type for the operation mode, too.
```javascript
/** @typedef {'normal' | 'simplified'} WorkbookOperationMode - The mode in which a workbook should be operated */
```

Reply:
Done (y)